guess.js

```diff
@@ -1,12 +1,14 @@
 //setting name game 
 let nameGame = "guess The Word";
+// [REVIEW] Why are you setting these HTML elements using JavaScript?

```

Comment:
Gotcha.
```suggestion
const nameGame = "guess The Word";
// [REVIEW] Why are you setting these HTML elements using JavaScript?

```
I'd then suggest making this a `const` since its value doesn't update.